backend/setupRedirects.js

```diff
@@ -22,5 +22,25 @@ module.exports = () => {
     res.redirect(301, `/prisoner/${req.params.offenderNo}/incentive-level-details`)
   )
 
+  if (config.app.featureFlags.redirectToBookingVideoLinkEnabled) {
+    const bvlRoutes = [
```

Comment:
So the idea would be to move this into the new `setupBvlRoutes.js`
```js
if (enabled) {
  // just add redirect for `/videolink` on its own
} else {
 // register all routes
}
```
When we remove vlb entirely from here we might want to semi-permanently add the `/videolink` redirect here.

Reply:
i see.